### DISCUSSION TITLE
Are there any plans to expand the package?

### DISCUSSION THREAD

**Discussion by mjahammel** [entechlog/dbt-snow-mask#60] [Q&A]
.. specifically, support for things like: creating/maintaining tags, creating/maintaining row access policies, assignment of tags, row access and masking policies at the table level in addition to the column level?

Thanks,

**Comment by entechlog** [comment]
@mjahammel,

We can definitely work to enhance the package, but would need help from the community.

**Comment by mjahammel** [comment]
I'm up for contributing.  Any suggestions as to where to start?  Look at current issues/pull requests, or my suggested extensions?
